src/controllers/v1.controller.ts

```diff
@@ -140,6 +140,13 @@ export class V1Controller {
 
       rpcID = parseRPCID(parsedRawData)
 
+      const [blockchainRequest] = this.host.split('.')
+
+      logger.log('info', `PUBLIC RPC RELAY REDIRECTED FOR ${blockchainRequest}`, {
```

Comment:
Ideally this should be accompanied by a test for logger output. But that is not a blocker IMO.